pkgs/by-name/sb/sbd/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sbd";
+  version = "1.37";
+
+  src = fetchurl {
+    url = "https://web.archive.org/web/20220626230420/http://www.tigerteam.se/dl/sbd/sbd-${finalAttrs.version}.tar.gz";
+    hash = "sha256-Qt5Q3PynV80DAXxuVqx6L4dTKgjnBDQjzSDZEf1kX10=";
+  };
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+    "CC=cc"
+  ];
+
+  buildFlags = lib.optional stdenv.isLinux "unix"
+               ++ lib.optional stdenv.isDarwin "darwin"
+               ++ lib.optional stdenv.hostPlatform.isWindows "win32";
+
+  meta = with lib; {
+    description = "Netcat-clone, portable, offers strong encryption - features AES-128-CBC + HMAC-SHA1 encryption, program execution, choosing source port, continuous reconnection with delay and more";
+    mainProgram = "sbd";
+    homepage = "https://packetstormsecurity.com/UNIX/netcat/";
+    license = licenses.gpl2Plus;
+    maintainers = with maintainers; [ d3vil0p3r ];
+    platforms = platforms.unix;
```

```suggestion
    platforms = platforms.unix;
```
this needs to get changed or `buildFlags` needs to get changed. have you tried to build for win32.
I tried to build `pkgsCross.mingwW64.sbd ` and `pkgsCross.mingw32.sbd` fail to build.

---

the home page should be `https://web.archive.org/web/20220626230420/http://www.tigerteam.se/dl/sbd/`

---

`-mno-cygwin` is deprecated and it should be removed. After that, for Windows, it is needed to find a way to include windows libraries like `winsock.h`.